### PR TITLE
some updates to the slack integration docs.

### PR DESCRIPTION
**Pull request by irabinovitch**
Some of the Slack instructions are a bit out of date.

### CODE REVIEW

File: content/integrations/slack.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+# Overview
 
+Connect Slack to Datadog in order to help your team collaborate by:
 
-* In your Slack account Administration page, go to the Integration tab and choose Datadog.
-* Copy paste the service hook into the field below.
-* Add the channels you want to be able to post to.
-* Tick the checkbox if you want to be notified for every comment, otherwise you will need to use the @slack-channel_name syntax.
+* Sharing graphs with your colleagues in your team's channels.
+* Receiving alerts and notifications from Datadog with in Slack.
```
Comment (kmshultz) [inline content/integrations/slack.md]: s/with in/within

File: content/integrations/slack.md [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+
+
+# Installation
+
+Slack is installed via its [integration tile](https://app.datadoghq.com/account/settings#integrations/slack) in the Datadog application.
```
Comment (kmshultz) [inline content/integrations/slack.md]: Prefer active voice: "Install Slack integration via", not "Slack is installed via"

File: content/integrations/slack.md [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+
+# Configuration
+1. In your Slack account  go to the [Applications page](https://slack.com/apps) and search for Datadog.
```
Comment (kmshultz) [inline content/integrations/slack.md]: > account  go

one too many whitespace.

File: content/integrations/slack.md [hunk 4]
```diff
@@ -1,13 +1,32 @@
...
+1. Click Install, followed by Add Integration.
+1. Copy the Slack Service Hook and paste in the [service hook field for Slack in Datadog](https://app.datadoghq.com/account/settings#integrations/slack).
+1  Add the channels you want to be able to post to.
+1. If you would like to be notified for every comment on a graph tick the check box "Transfer all user comments" by each channel. If left unchecked (default) you will need to use the @slack-channel_name syntax for comments to be posted to slack.
```
Comment (kmshultz) [inline content/integrations/slack.md]: insert comma: "for every comment on a graph, tick the check box"

s/slack/Slack

**Comment by irabinovitch** [comment]
@kmshultz all set

**Pull request merged**